var/spack/repos/builtin/packages/httpie/package.py

```diff
@@ -0,0 +1,39 @@
+##############################################################################
+# Copyright (c) 2013-2016, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/llnl/spack
+# Please also see the LICENSE file for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+
+
+class Httpie(PythonPackage):
+    """Modern command line HTTP client."""
+
+    homepage = "https://httpie.org/"
+    url      = "https://github.com/jakubroztocil/httpie/archive/0.9.8.tar.gz"
+
+    version('0.9.8', '425a1c92590d9b906a7369f5c17990a4')
+
+    depends_on('py-setuptools',   type=('build', 'run'))
+    depends_on('py-pygments',     type=('build', 'run'))
```

Comment:
Yep, the setup.py is what actually tells the package what it requires. It contains all you need to know. Unless the package doesn't use setuptools, in which case you kinda have to use trial and error. I don't think distutils let's you specify dependencies.